docs/change_log/index.md

```diff
@@ -6,6 +6,7 @@ Python-Markdown Change Log
 (under development): version 3.3.7 (a bug-fix release).
 
 * Disallow square brackets in reference link ids (#1209).
+* Support custom CSS classes on TOC element (#1224).
```

Comment:
Version 3.3.7 is a point release and there can only contain bug fixes. It should not include new features. See the [Contributing Guide][1] for details. In any event, this will need to be part of the 3.4.0 release.

[1]: https://python-markdown.github.io/contributing/#versions